Add binary file debdiff line number (#122)

cpina opened the pull request:
Suggestion for #122

cpina committed:
Binary file: which line in the diff file? (unit test without implementation)

cpina committed:
PatchSet._parse(): pass diff_line_no to PatchedFile

SuperMeepEnby commented:
Hey @cpina !

This repository seems unmaintained. I started a new repository where I intend on maintaining it. I dropped support for Python2, added a CI as well as type annotations (and had to fix some). I also integrated a PR suggested here to fix diff with Git mnemonicPrefix option enabled.

I'd like to take a look at your suggestion on this new repository, which you can find here: https://github.com/SuperMeepEnby/python-unidiff2.

cpina commented:
> Hey @cpina !
> 
> This repository seems unmaintained. I started a new repository where I intend on maintaining it. I dropped support for Python2, added a CI as well as type annotations (and had to fix some). I also integrated a PR suggested here to fix diff with Git mnemonicPrefix option enabled.
> 
> I'd like to take a look at your suggestion on this new repository, which you can find here: https://github.com/SuperMeepEnby/python-unidiff2.

Done. Good luck with the fork!
